tribe-ext-calendar-widget-areas/index.php

```diff
@@ -0,0 +1,280 @@
+<?php
+/**
+* Plugin Name: The Events Calendar Extension: Calendar Widget Areas
+* Description: Adds widget areas (a.k.a. sidebars) that only display on The Events Calendar pages/views. Areas may be enabled or disabled in the Display tab of Events Settings.
+* Version: 1.0
+* Extension Class: Tribe__Extension__Calendar_Widget_Areas
+* Author: Modern Tribe, Inc.
+* Author URI: http://m.tri.be/1971
+* License: GPLv2 or later
+* License URI: https://www.gnu.org/licenses/gpl-2.0.html
+*/
+
+// Created 2015-10-05 for https://theeventscalendar.com/support/forums/topic/control-visibility-of-sidebar-widgets/#dl_post-1011743
+// Turned into an Extension by Cliff on 2016-12-23
+
+// Do not load directly.
+if ( ! defined( 'ABSPATH' ) ) {
+	die( '-1' );
+}
+
+// Do not load unless Tribe Common is fully loaded.
+if ( ! class_exists( 'Tribe__Extension' ) ) {
+	return;
+}
+
+// Do not load unless dynamic_sidebar exists -- since WP 2.2.0, per https://developer.wordpress.org/reference/functions/dynamic_sidebar/
+if ( ! function_exists( 'dynamic_sidebar' ) ) {
+	return;
+}
+
+/**
+* Extension main class, class begins loading on init() function.
+*/
+class Tribe__Extension__Calendar_Widget_Areas extends Tribe__Extension {
+	/**
+	 * Option key for which widget areas are enabled
+	 *
+	 * @var string
+	 */
+	protected $option_key_enabled_areas = 'tribe_ext_enabled_widget_areas';
+
+	/**
+	 * Setup the Extension's properties.
+	 *
+	 * This always executes even if the required plugins are not present.
+	 * 
+	 * @access: public
+	 */
+	public function construct() {
+		// Each plugin required by this extension
+		$this->add_required_plugin( 'Tribe__Events__Main' );
+				
+		// Set the extension's TEC URL
+		$this->set_url( 'https://theeventscalendar.com/extensions/calendar-widget-areas/' );
+	}
+	
+	/**
+	 * All available widget areas this extension supports and details about each in a multidimensional associative array.
+	 *
+	 * Follow this format to enable a new widget area for each action hook you want
+	 * to use. You will also need to add an additional method toward the end of this
+	 * file for each area added so we know what code to execute.
+	 * 
+	 * @access: public
+	 * @return: array
```

Comment:
For these to be parsed properly there should not be a colon after the property. so just `@return array` would be perfect. I would do this in all of your docblocks.